### PR TITLE
Increase pytest coverage

### PR DESCRIPTION
**Pull request by aliceinwire**

### CODE REVIEW

File: tests/test_libs_and_subcommands_detail.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+def test_load_toml_missing_required_config_aborts(tmp_path, monkeypatch):
+    monkeypatch.chdir(tmp_path)
+    monkeypatch.setenv("HOME", str(tmp_path / "home"))
+
+    with pytest.raises(click.Abort):
+        load_toml("missing.toml", "checkout")
+
+
+def test_config_command_creates_private_example_file(tmp_path):
```
Comment (bhcopeland) [inline tests/test_libs_and_subcommands_detail.py]: I would probably isolate this to HOME. I.e. when I tested this locally I got "Config file already present", since its already there. I see you use HOME on test_load_toml_missing_required_config_aborts, so probably we should here too?

File: tests/test_libs_and_subcommands_detail.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    state = dict(bisect.default_state, giturl="https://git.example/linux.git")
+
+    assert bisect.load_state(str(path)) is None
+    bisect.save_state(state, str(path))
+    assert bisect.load_state(str(path)) == state
+
+
+def test_commit_find_diff_returns_latest_patch(tmp_path):
+    repo = subprocess.run(
+        ["git", "init", str(tmp_path)], check=True, capture_output=True, text=True
```
Comment (bhcopeland) [inline tests/test_libs_and_subcommands_detail.py]: subprocess.run(["git", "init", "-b", "master", str(tmp_path)], ...)

I would force master here, because later down you use master, and there is no enforcement, so its a edge case if it ever creates a repo on main instead of master.

File: tests/test_libs_and_subcommands_detail.py (hunk 1 above)
Comment (aliceinwire) [reply tests/test_libs_and_subcommands_detail.py]: fixed

File: tests/test_libs_and_subcommands_detail.py (hunk 2 above)
Comment (aliceinwire) [reply tests/test_libs_and_subcommands_detail.py]: fixed
